ledger/acctupdates.go

```diff
@@ -1140,6 +1143,9 @@ func (au *accountUpdates) lookupWithoutRewards(rnd basics.Round, addr basics.Add
 		// a separate transaction here, and directly use a prepared SQL query
 		// against the database.
 		persistedData, err = au.accountsq.lookup(addr)
+		if err != nil {
+			return ledgercore.AccountData{}, basics.Round(0), rewardsVersion, rewardsLevel, err
+		}
 		if persistedData.round == currentDbRound {
 			if persistedData.rowid != 0 {
 				// if we read actual data return it
```

Comment:
Should we no explicitly return `nil`, instead of `err` here?

Reply:
This comment got attached to the wrong place.  I'm referring to some code that was not touched, which returns `err` a few lines later.  But by that point, `err` must be nil.

Reply:
to which code? 
>			// otherwise return empty
?

Reply:
These two uses of `err` here could be `nil` because they come after your new `err` check, and with no intervening assignments to `err`.
```
				return persistedData.accountData.GetLedgerCoreAccountData(), rnd, rewardsVersion, rewardsLevel, err
			}
			// otherwise return empty
			return ledgercore.AccountData{}, rnd, rewardsVersion, rewardsLevel, err
```